### ISSUE TITLE
How to use with Auth module and Google provider

### ISSUE THREAD

**Issue by learntheropes** [motia/nuxt-auth-refresh#5]
Cai I use it with the Auth module and Google provider?
Would you mind to provide an example?

**Comment by motia** [comment]
I have added the feature in the dev branch. You can try it out, though I did not try it with OAuth yet.
To provide an example, I am afraid I can not do that in the near future.

But you can try the feature yourself:
```
yarn add https://github.com/motia/nuxt-auth-refresh@dev
```

Basically, you need is to configure the package as follows:
```
{
      authStrategy: 'google',
      authScheme: 'oauth2',
      loginUrl: 'www.googleapis.com/oauth2/v4/token',
      refreshUrl: 'www.googleapis.com/oauth2/v4/token',
      logoutUrl: 'https://accounts.google.com/o/oauth2/revoke',

      refreshPeriod: X // a number of seconds that is adequate for the Google provider
}
```
Also, make sure the auth module is configured correctly. You should set the `authorization_endpoint` to 'https://accounts.google.com/o/oauth2/auth'

https://auth.nuxtjs.org/providers/google.html
https://github.com/nuxt-community/auth-module/blob/dev/lib/providers/google.js

Cheers

**Comment by motia** [comment]
merged into master #8
